movie-search.js

```diff
@@ -18,9 +21,10 @@ const printTitles = (rawData) => {
   titles.forEach((element) => {
     console.log(element)
   })
+  return titles
```

Comment:
@bundacia What's the difference between `titles.forEach(console.log)` and `titles.forEach(element => console.log(element))`? For some reason, the first way logs: 
```
 Inception (2010)  0 [ ' Inception (2010) ',
  ' Inception (2010) (TV Episode)  - Season 4 | Episode 7  - Elevator (2007) (TV Series)  ',
  ' Inception (2010) (TV Episode)  - Season 1 | Episode 14  - The Distressed Watcher (2010) (TV Series)  ',
  ' Inception: The Cobol Job (2010) (Video) ' ]
 Inception (2010) (TV Episode)  - Season 4 | Episode 7  - Elevator (2007) (TV Series)   1 [ ' Inception (2010) ',
  ' Inception (2010) (TV Episode)  - Season 4 | Episode 7  - Elevator (2007) (TV Series)  ',
  ' Inception (2010) (TV Episode)  - Season 1 | Episode 14  - The Distressed Watcher (2010) (TV Series)  ',
  ' Inception: The Cobol Job (2010) (Video) ' ]
 Inception (2010) (TV Episode)  - Season 1 | Episode 14  - The Distressed Watcher (2010) (TV Series)   2 [ ' Inception (2010) ',
  ' Inception (2010) (TV Episode)  - Season 4 | Episode 7  - Elevator (2007) (TV Series)  ',
  ' Inception (2010) (TV Episode)  - Season 1 | Episode 14  - The Distressed Watcher (2010) (TV Series)  ',
  ' Inception: The Cobol Job (2010) (Video) ' ]
 Inception: The Cobol Job (2010) (Video)  3 [ ' Inception (2010) ',
  ' Inception (2010) (TV Episode)  - Season 4 | Episode 7  - Elevator (2007) (TV Series)  ',
  ' Inception (2010) (TV Episode)  - Season 1 | Episode 14  - The Distressed Watcher (2010) (TV Series)  ',
  ' Inception: The Cobol Job (2010) (Video) ' ]
```

and the second way logs the expected: 

```
 Inception (2010)
 Inception (2010) (TV Episode)  - Season 4 | Episode 7  - Elevator (2007) (TV Series)
 Inception (2010) (TV Episode)  - Season 1 | Episode 14  - The Distressed Watcher (2010) (TV Series)
 Inception: The Cobol Job (2010) (Video)
```